contracts/BitmaskRBAC.sol

```diff
@@ -35,20 +40,26 @@ contract BitmaskRBAC is RBAC {
   address[] public userList;
 
   mapping(string => bool) supportedRoles;
+  mapping(string => uint) rolePosition;
   string[] public supportedRoleList;
 
   mapping(string => uint) userCountsByRole;
+  event RoleAdded(address indexed operator, string role);
```

Comment:
I was looking at a more recent version than the one we were inheriting from.  Our TruSetPortal switches on event name, not event signature, so it should be fine.  I think changing `RoleAdded` -> `RoleGranted` would create an unnecessary need to handle both event names in our clients (since we have contracts out in the wild).  I don't think thats worthwhile.

https://github.com/OpenZeppelin/openzeppelin-solidity/pull/981/files added indexed parameters (and woah it's a huge PR)

I'm fine with you taking out the indexed parameters, but I think they should stay.

Reply:
As discussed, fine to keep these if the webapp still reads the log value (was a field, now a topic) successfully.